### PR TITLE
fix: add AVI-SHUKLA1110

### LINKED ISSUE #1732: [PROFILE ADDITION] <Abhishek Shukla>
### Profile screenshot

Screenshot 2023-02-16 195412

### PR DESCRIPTION
**Pull request by AVI-SHUKLA1110**
resolve #1732

resolves #issueNumber

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: AVI-SHUKLA1110.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/88367072/219391424-f79f32ac-0b3f-42ad-a19c-3fb0cfacb186.png",
+  "issueId": 66,
```
Comment (Panquesito7) [inline AVI-SHUKLA1110.json:3]: Wrong issue ID.
```suggestion
  "issueId": 1732,
```

File: AVI-SHUKLA1110.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/88367072/219391424-f79f32ac-0b3f-42ad-a19c-3fb0cfacb186.png",
```
Comment (Panquesito7) [inline AVI-SHUKLA1110.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/219433518-19a7d654-552c-4cee-a2a0-3153108ed35e.png",
```

**Comment by Panquesito7** [comment]
Also, this seems to be a duplicate of #1707. 🤔

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by AVI-SHUKLA1110** [comment]
thnks @Pradumnasaraf :)

**Pull request merged**